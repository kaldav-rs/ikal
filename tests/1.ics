BEGIN:VCALENDAR
PRODID:-//Nextcloud calendar v1.5.0
VERSION:2.0
CALSCALE:GREGORIAN
BEGIN:VEVENT
CREATED:20170209T192358
DTSTAMP:20170209T192358
LAST-MODIFIED:20170209T192358
UID:5UILHLI7RI6K2IDRAQX7O
SUMMARY:Vers
CLASS:PUBLIC
STATUS:CONFIRMED
DTSTART;VALUE=DATE-TIME;TZID=Europe/Paris:20150219T190000
DTEND;VALUE=DATE:20170210
END:VEVENT
END:VCALENDAR
