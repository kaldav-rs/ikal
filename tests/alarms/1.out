Ok(
    Audio(
        Audio {
            action: "AUDIO",
            trigger: DateTime(
                1997-03-17T14:30:00+01:00,
            ),
            duration: Some(
                Duration {
                    secs: 900,
                    nanos: 0,
                },
            ),
            repeat: Some(
                4,
            ),
            attach: [
                "ftp://example.com/pub/sounds/bell-01.aud",
            ],
            x_prop: {},
            iana_prop: {},
        },
    ),
)
